src/storage/v2/edge_accessor.hpp

```diff
@@ -86,6 +86,8 @@ class EdgeAccessor final {
   Result<std::map<PropertyId, PropertyValue>> Properties(View view) const;
 
   Gid Gid() const noexcept;
+  auto GidPropertiesOnEdges() const { return edge_.ptr->gid; }
```

Comment:
```suggestion
  auto GidPropertiesOnEdges() const -> Gid { return edge_.ptr->gid; }
```

Reply:
:+1: 
Had to move `Gid Gid() const noexcept;` down because it shadowed Gid the type.